tools/watermill-cli/README.md

```diff
@@ -0,0 +1,62 @@
+# CLI Wrapper for Watermill
+
+This is a CLI wrapper for Watermill. It has two basic functionalities: producing and consuming messages on the following
+Pub/Subs:
+
+1. Kafka
+2. Google Cloud Pub/Sub
+3. RabbitMQ
+
```

Comment:
Installation guide is missing

Reply:
done b3ae9f252ccb6e90efcf9f72512765a9df71cff4